packages/react/src/components/Masthead/MastheadL1.js

```diff
@@ -20,19 +21,31 @@ const { prefix } = settings;
  * @typedef {object} navigation Object containing navigation elements
  * @returns {*} Masthead component
  */
-const MastheadL1 = () => {
+const MastheadL1 = ({ isShort }) => {
   const className = cx({
     [`${prefix}--masthead__l1`]: true,
+    [`${prefix}--masthead__l1--short`]: isShort,
+  });
+
+  const nameShortL1 = cx({
+    [`${prefix}--masthead__l1-name--shortl1`]: isShort,
+  });
+  const eyebrowShortL1 = cx({
+    [`${prefix}--masthead__l1-name-eyebrow--shortl1`]: isShort,
+  });
+  const titleShortL1 = cx({
+    [`${prefix}--masthead__l1-name-title--shortl1`]: isShort,
   });
 
   return (
     <div className={className}>
-      <div className={`${prefix}--masthead__l1-name`}>
-        <span className={`${prefix}--masthead__l1-name-eyebrow`}>
+      <div className={`${prefix}--masthead__l1-name ${nameShortL1}`}>
+        <span
+          className={`${prefix}--masthead__l1-name-eyebrow ${eyebrowShortL1}`}>
```

Comment:
@emerson-pereira going to leave my comment about the eyebrow shrinking here since it'll be easier to form a thread this way.

From my understanding of the ticket: https://github.com/carbon-design-system/ibm-dotcom-library/issues/146
`When the eyebrow is active then leave the L1 at 80px, fixed to the top, just like it is right now.` 
It seems we don't shrink L1 to 48px for all cases, if there is an eyebrow (it is hardcoded as of now, you can set it up so it is passed through as a prop - that would help with testing) we don't want to shrink it 

@larahanlon2 can you clarify?

Reply:
@jeffchew @emerson-pereira cross out what I said 😅we do always shrink to 48px

from @larahanlon2:
> - we have the eyebrow in the L1 - L1 height will be 80px when not sticky, but will shrink to 48px when sticky (eyebrow is hidden)
> - we don’t have the eyebrow in L1 - L1 height will be 80px when not sticky, but will shrink to 48px when sticky